webui/ocw/views.py

```diff
@@ -45,6 +47,12 @@ def update_status(request):
     return redirect('instances')
 
 
+def cron(request):
```

Comment:
I would say it should require login

Reply:
So will something periodically trigger this function or it is just confusing name ? In second case suggesting to rename it because I am expecting that something with such name should have auto-triggering infra